tests/mock_trello.py

```diff
@@ -1,46 +1,109 @@
-from unittest.mock import patch, MagicMock
-from status import NOT_STARTED, COMPLETED
-from item import Item
-from config import TRELLO_URL, TRELLO_BOARD
-import sys
-import pytest
+from unittest.mock import MagicMock
+from api_client import ApiClient
+from status import NOT_STARTED, COMPLETED, IN_PROGRESS
+from trello import Trello
 
-requestsMock = MagicMock()
+def mock_get(response_body, status_code=200):
```

Comment:
This function name isn't very clear. It's building a mock response. Nothing about this function is related to GET requests.